styles/bsky/catppuccin.user.less

```diff
@@ -582,6 +582,10 @@
       background-color: @overlay2 !important;
     }
 
+    path[stroke*="hsl(211, 99%, 56%)"] {
+      stroke: @accent !important;
+    }
+
```

Comment:
could you add a comment here saying that thats for the character limit progress circle? as reference for future fixes :3